virttest/env_process.py

```diff
@@ -1466,6 +1471,10 @@ def postprocess(test, params, env):
 
     if err:
         raise RuntimeError("Failures occurred while postprocess:\n%s" % err)
+    elif (params.get("setup_hugepages") == "yes" and
+          post_hugepages_surp > pre_hugepages_surp):
+        leak_num = post_hugepages_surp - pre_hugepages_surp
+        raise exceptions.TestWarn("%d huge pages leaked!" % leak_num)
```

Comment:
changes in this file is not necessary.

Reply:
Yes, maybe. I just want vt to report `WARN` status instead of `PASS`.